index.html

```diff
@@ -258,7 +258,7 @@ <h1 class="features-title">How can you join?</h1>
             put in any efforts for yourself. We want to value everyone’s time
             and efforts.
           </p>
-          <button class="btn-join" id="btn-join">Join the Squad</button>
+          <a class="btn-join" href="#info-note">Join the Squad</a>
```

Comment:
This is not visible in the production until user puts the feature flag, so we can move ahead with the button